tsconfig.json

```diff
@@ -16,5 +16,5 @@
         "noUnusedLocals": true,
         "strictNullChecks": true
     },
-    "exclude": ["**/*.d.ts"]
+    "exclude": ["**/dist/**/*", "node-modules/**/*"]
```

Comment:
"dist/**/*" ?
I think "**/dist/**/*" and "dist/**/*"  are equivalent but this is not consistent with "node-modules/**/*"

Reply:
updated to how it was mened